src/shared/ui-kit/components/FormTokenInput/FormTokenInputSelectTokenModal.tsx

```diff
@@ -115,13 +118,13 @@ export function FormTokenInputSelectTokenModal() {
               minWidth="100%"
             />
           </InputGroup>
-          {isEmpty(tokens) ? (
+          {isEmpty(nonNFTTokens) ? (
             <Flex alignItems="center" justifyContent="center" width="100%" height="100%" maxHeight="296px">
               <Text textStyle="p medium semibold">No results found.</Text>
             </Flex>
           ) : (
             <List variant="select-token-list">
-              <FixedSizeList height={296} itemCount={tokens.length} itemSize={64} width="100%">
+              <FixedSizeList height={296} itemCount={nonNFTTokens.length} itemSize={64} width="100%">
                 {({ index }) => <TokenListItem key={index} index={index} />}
               </FixedSizeList>
             </List>
```

Comment:
<img width="475" alt="Screenshot 2023-08-31 at 11 18 58 AM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/89677167/819e819d-6d3e-4bcd-87f0-0e334a5a6b45">
we could show some error that "do not have sufficient balance"

Reply:
Done.